cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -324,6 +325,7 @@ protected void configure() {
       handlerBinder.addBinding().to(ApplicationTemplateHandler.class);
       handlerBinder.addBinding().to(WorkflowHttpHandler.class);
       handlerBinder.addBinding().to(ArtifactHttpHandler.class);
+      handlerBinder.addBinding().to(WorkflowStatsSlaHttpHandler.class);
```

Comment:
Should be `SLA`, not `Sla`. Naming convention for known acronym should be all caps (e.g. `URL`, `URI`, `UUID`, ...)